actors/paych/src/lib.rs

```diff
@@ -132,7 +132,11 @@ impl Actor {
             None,
             SendFlags::READ_ONLY,
         ))
-        .map_err(|e| e.wrap("voucher sig authentication failed"))?;
+        .and_then(deserialize_block)
+        .unwrap_or(false)
+        {
+            return Err(actor_error!(forbidden, "voucher sig authentication failed"));
```

Comment:
I'm discarding the original exit code because we really should be failing with "forbidden" here.

Reply:
No, definitely not what I want to do here.